### PR TITLE
feat: UI for unsupported message

### PR DESCRIPTION
**Pull request by scmmishra**
localhost_3000_app_accounts_1_conversations_8504 (1)
localhost_3000_app_accounts_1_conversations_8504

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by scmmishra** [comment]
@pranavrajs need your thoughts on the UI and copy
CC: @nithindavid @absurdiya

**Comment by absurdiya** [comment]
> @pranavrajs need your thoughts on the UI and copy CC: @nithindavid @absurdiya

We should use a warning/error color palette, this seems very similar to private note (which is also yellow)

File: app/javascript/dashboard/i18n/locale/en/conversation.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "SAVE_CONTACT": "Save",
     "UPLOADING_ATTACHMENTS": "Uploading attachments...",
     "REPLIED_TO_STORY": "Replied to your story",
+    "UNSUPPORTED_MESSAGE": "This message is unsupported. You can view this message on the mobile app.",
```
Comment (sojan-official) [inline app/javascript/dashboard/i18n/locale/en/conversation.json:44]: lets also tweak this text little bit.  view this on mobile app must be confusing to the user as they would imagine the mobile app to be chatwoot. lets change this to reflect that they need to view this message on facbook/instagram

**Comment by scmmishra** [comment]
> > @pranavrajs need your thoughts on the UI and copy CC: @nithindavid @absurdiya
> 
> We should use a warning/error color palette, this seems very similar to private note (which is also yellow)

@absurdiya updated, PTAL

**Comment by absurdiya** [comment]
> > > @pranavrajs need your thoughts on the UI and copy CC: @nithindavid @absurdiya
> > 
> > 
> > We should use a warning/error color palette, this seems very similar to private note (which is also yellow)
> 
> @absurdiya updated, PTAL

Looks good to me

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**